webrob/pages/admin/db.py

```diff
@@ -145,3 +160,80 @@ def db_remove_route(table):
     if data is not None and 'id' in data and db_find(cls, data['id']):
         db_remove(cls, data['id'])
     return jsonify(result=None)
+
+
+@app.route('/edit_neem_hub_settings')
+@admin_required
+def render_neem_hub_settings_page_get():
+    # PasswordForm used for validating given password field
+    form = PasswordForm()
+    try:
+        neemHubSettings = get_settings()
+    except SQLAlchemyError as e:
```

Comment:
why not adding a flask handler for this error?

Reply:
done. Now SQLAlchemyError has its own flask handler.